ScoutHelper/Managers/TerritoryManager.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using CSharpFunctionalExtensions;
+using Dalamud;
+using Dalamud.Plugin.Services;
+using Lumina.Excel.GeneratedSheets2;
+using ScoutHelper.Config;
+using ScoutHelper.Utils;
+using ScoutHelper.Utils.Functional;
+
+namespace ScoutHelper.Managers;
+
+public class TerritoryManager {
+	private static readonly ISet<string> SupportedMaps = new HashSet<string>() {
+		// HW
```

Comment:
Rip arr

Reply:
XD. i do need to add it in, but in my rip of siren months ago i left out arr because of a bunch of annoyances XD. will add it back in an update :P

Reply:
Yeah I remember arr having a problem since the encoding didn't work for one of the maps (probably coerthas central highlands). I assumed that (and probably some other annoyances I didn't know about) was the reason they were skipped